test/test_fx_experimental.py

```diff
@@ -1459,6 +1459,9 @@ class TestNormalizeOperators(JitTestCase):
     def test_normalize_operator_exhaustive(self, device, dtype, op):
         # Sorted and one entry on each line to minimize merge conflicts.
         op_skip = {
+            # See: https://github.com/pytorch/pytorch/issues/64997
```

Comment:
FYI @zou3519 that might cause issues for functorch as well?

Reply:
Thanks for the heads up! 

To clarify, is the potential problem that adding these two new OpInfos might break functorch tests?

Reply:
Yes.

Reply:
We should get rid of this skip list in favor of annotating opinfos themselves with the appropriate metadata for fx (or functorch) to infer that their systems/tests won't work.